statsmodels/gam/gam.py

```diff
@@ -0,0 +1,767 @@
+# -*- coding: utf-8 -*-
+"""
+Generalized Additive Models
+
+Author: Luca Puggini
+Author: Josef Perktold
+
+created on 08/07/2015
+"""
+
+from __future__ import division
+import numpy as np
+from scipy import optimize
+from statsmodels.discrete.discrete_model import Logit
+from statsmodels.genmod.generalized_linear_model import (GLM, GLMResults,
+    GLMResultsWrapper, lm, _check_convergence)
+import statsmodels.regression._tools as reg_tools
+from statsmodels.tools.sm_exceptions import PerfectSeparationError
+from statsmodels.tools.decorators import cache_readonly
+from statsmodels.base._penalized import PenalizedMixin
+from statsmodels.gam.gam_penalties import MultivariateGamPenalty
+from statsmodels.tools.linalg import matrix_sqrt
+
+
+class GLMGAMResults(GLMResults):
+    """Results class for generalized additive models, GAM.
+
+    This inherits from GLMResults.
+
+    Warning: not all inherited methods might take correctly account of the
+    penalization
```

Comment:
the phrasing of this sentence is awkard.  maybe "some inherited methods might not correctly take account of the penalization"?